src/ShallowTraversal.js

```diff
@@ -27,6 +27,9 @@ export function childrenOfNode(node) {
 
 export function hasClassName(node, className) {
   let classes = propsOfNode(node).className || '';
+  if (typeof classes !== 'string') {
+    classes = classes.toString();
+	}
```

Comment:
If React stringifies it without a warning, then I'm ok with us doing it too (using `String()`, never `.toString`). Otherwise, yes, this is the explicit responsibility of the user (i.e. option 2)